File: Unit2-06/Extra/Airplane.java
/*
* This program is the standard airplane class.
*
* @author  Samuel Webster
* @version 1.0
* @since   2024-03-25
*/

/**
* This is the airplane class.
*/
public class Airplane {
    /** The internal variable that stores speed. */
    private int speed;

    /**
    * Constructor.
    *
    * @param args no arguments
    */
    public Airplane(String[] args){
    }

    /**
    * Set an airplane speed.
    *
    * @param speed set speed
    */
    public void setSpeed(int speed) {
        this.speed = speed;
    }

    /**
    * Get speed.
    *
    * @param args no arguments
    */
    public int getSpeed(String[] args) {
        return speed;
    }
}

File: Unit2-06/Airplane.ts
/*
* This program is the standard airplane class.
*
* @author  Samuel Webster
* @version 1.0
* @since   2024-03-25
*/


// This is the airplane class.
export class Airplane {
    /** The internal variable that stores speed. */
    private speed: number

    // Constructor.
    public Airplane() {
    }

    // Set an airplane speed.
    public setSpeed(speed: number) {
        this.speed = speed
    }

    // Get speed.
    public getSpeed(): number {
        return this.speed
    }
}
